trio/_tools/gen_exports.py

```diff
@@ -310,6 +315,17 @@ def main() -> None:  # pragma: no cover
 
 """
 
+IMPORTS_WINDOWS = """\
+from typing import TYPE_CHECKING, ContextManager
+
+if TYPE_CHECKING:
+    from .._file_io import _HasFileNo
```

Comment:
Seems iffy to be importing a private name from a different module, might want to make this also public.